staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -141,6 +143,9 @@ type GenericAPIServer struct {
 	healthzLock    sync.Mutex
 	healthzChecks  []healthz.HealthzChecker
 	healthzCreated bool
+
+	// Endpoints routing
+	EndpointRouter *service.REST
```

Comment:
actually, this is something that only the core apiserver will be able to provide, right? not sure why it's in this file